AddUserToNodeGroup/index.ts

```diff
@@ -0,0 +1,77 @@
+import { AzureFunction, Context, HttpRequest } from '@azure/functions';
+import { ValidationError } from 'joi';
+import addUserToNodeGroupSchema from './req.validation';
+import getConnection from '../shared/mongoose';
+import NodeGroup from '../shared/nodeGroup/nodeGroup.interface';
+import NodeGroupModel from '../shared/nodeGroup/nodeGroup.model';
+import UserModel from '../shared/user/user.model';
+import { User, Role } from '../shared/user/user.interface';
+import authWrapper from '../shared/authWrapper';
+import HttpTriggerResult from '../shared/httpTriggerResult';
+import {
+    ForbiddenFunctionError,
+    httpErrorHandler,
+    ValidationFunctionError,
+    ObjectNotFoundFunctionError,
+    ObjectAlreadyExistInArrayFunctionError,
+} from '../shared/error/error';
+
+const functionName = 'addUserToNodeGroup';
+
+const addUserToNodeGroup: AzureFunction = async (context: Context, req: HttpRequest, user: User): Promise<HttpTriggerResult> => {
+    try {
+        if (user.role !== Role.RamadIturOfUnit) {
+            throw new ForbiddenFunctionError(functionName, user);
+        }
+        const validationError: ValidationError = addUserToNodeGroupSchema.validate(req).error;
+        if (validationError) {
+            throw new ValidationFunctionError(functionName, validationError.message);
+        }
+
+        const { nodeGroupId } = context.bindingData;
+        const { userId } = context.bindingData;
+
+        await getConnection();
+
+        const nodeGroup: NodeGroup | void = await NodeGroupModel.findOne({ id: nodeGroupId }).exec();
+        if (!nodeGroup) {
+            throw new ObjectNotFoundFunctionError(functionName, 'nodeGroup', nodeGroupId);
+        }
+
+        const userToAdd: User | void = await UserModel.findById(userId).exec();

```

Comment:
Change to identity number
```suggestion
        const userToAdd: User = await UserModel.findById(userId).exec();

```
Remove every `| void`